backend/src/sms/middlewares/sms.middleware.ts

```diff
@@ -153,22 +145,58 @@ export const InitSmsMiddleware = (
     const logMeta = {
       campaignId,
       label,
-      action: 'getCredentialsFromLabel',
+      action: 'getCredentialsFromLabelCampaign',
     }
     try {
       /* Determine if credential name can be used */
       const campaign = campaignId
         ? await SmsService.findCampaign(+campaignId, userId)
         : null
-      const credentialName = campaign?.demoMessageLimit
+      const canUseDemoCredentials = !!campaign?.demoMessageLimit // false if (1) campaign is null or (2) demoMessageLimit is 0
```

Comment:
Can we use campaign?.demoMessageLimit && campaign?.demoMessageLimit > 0 instead? I know this is equivalent of the existing code, but it's quite confusing